compiler/src/dotty/tools/dotc/transform/ReifyQuotes.scala

```diff
@@ -88,7 +88,8 @@ import dotty.tools.dotc.core.quoted._
 class ReifyQuotes extends MacroTransformWithImplicits with InfoTransformer {
   import ast.tpd._
 
-  private lazy val splicer: Splicer = new Splicer
+  /** Classloader used for loading macros */
+  private var macroClassLoader: java.lang.ClassLoader = _
```

Comment:
`private[this]` also usually we put the field and the getter next to each other, and we call the field `myXXX` and the getter `XXX`.